ui/index.jsx

```diff
@@ -105,13 +105,15 @@ OCVN.STYLING = {
   charts: {
     axisLabelColor: "#cc3c3b",
     traceColors: ["#234e6d", "#3f7499", "#80b1d3", "#afd5ee", "#d9effd"],
+    hoverFormat: ',.2f',
     hoverFormatter: number => {
       if(typeof number == "undefined") return number;
```

Comment:
maybe move format function into a util module would be good just in case it may be needed from some other charts or components